Code_Sample_README.md

```diff
@@ -0,0 +1,83 @@
+# blackbaud-crm-sky
```

Comment:
This README can take the place of the normal README. That way, it will be displayed prominently on the front page. The main code and the fork will always be separate, so this difference won't cause issues.